### PR TITLE
Ndev 2561 ci upgrade

### PR DESCRIPTION
**Pull request by kristinaNikolaevaa**
Added posibiliry to run basic tests on hetzner infrastructure
I will add it for economic and OZ tests but in next PR

https://github.com/neonlabsorg/neon-tests/actions/runs/8097395499

### CODE REVIEW

File: .github/workflows/basic.yml [hunk 1]
```diff
@@ -73,49 +65,35 @@ jobs:
...
-          echo "Network: ${NETWORK}"
-          echo "Runner: ${RUNNER}"
-          echo "Numprocesses: ${NUMPROCESSES}"
-          echo "Build url: ${{ env.BUILD_URL }}"
-
-          echo "network=${NETWORK}" >> $GITHUB_OUTPUT
-          echo "runner=${RUNNER}" >> $GITHUB_OUTPUT
-          echo "jobs=${JOBS_NUMBER}" >> $GITHUB_OUTPUT
-          echo "numprocesses=${NUMPROCESSES}" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
```
Comment (gigimon) [inline .github/workflows/basic.yml:68]: If we want run our tests on night-stand or devnet why we need to setup terraform?

Comment (kristinaNikolaevaa) [reply .github/workflows/basic.yml:68]: We have already disscussed it
1) it’s down very often
2) it’s updaiting only 1 time a day
3) The configuration is not the same as in proxy CI
4) we can't run tests on several versions (for now we have changes in 1.9.x and in develop)
5) we can't run operator tests in the middle of day on night stand (we can't be sure that nobody use it)

night-stand and devnet are good for night runs ,  not for runs for each PR

**Review by gigimon — APPROVED** [review]

File: envs.json [hunk 2]
```diff
@@ -97,7 +97,7 @@
     "network_ids": {
       "neon": 245022926
     },
-    "solana_url": "https://api.devnet.solana.com",
+    "solana_url": "https://devnet.sol-rpc.neoninfra.xyz:8443/qccxeCwY02UM5p0ELvbdHS9uedrKbPE2f7eUqabd/",
```
Comment (andreisilviudragnea) [inline envs.json:100]: @gigimon Was this change intended?
